UA-CloudPublisher.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
 	  <OutputType>Exe</OutputType>
-    <TargetFramework>net9.0</TargetFramework>
+    <TargetFramework>net10.0</TargetFramework>

```

Comment:
Moving to a completely new .Net framework version is a big change. Did you thoroughly test all features of Publisher before sending the PR?